src/main/java/it/gov/pagopa/ranking/dto/mapper/InitiativeBuild2ConfigMapper.java

```diff
@@ -34,6 +37,7 @@ public InitiativeConfig apply(InitiativeBuildDTO initiativeBuildDTO) {
                 .rankingStatus(RankingStatus.WAITING_END)
                 .size(calculateSize(initiativeBuildDTO))
                 .rankingFields(retrieveRankingFieldCodes(automatedCriteriaList))
+                .isLogoPresent((additionalInfo != null && additionalInfo.getLogoFileName() != null) ? Boolean.TRUE : Boolean.FALSE)
```

Comment:
you could simplify this expression
Moreover, using
`org.apache.commons.lang3.StringUtils#isEmpty`
you could also check if it is not null and not empty
```suggestion
                .isLogoPresent(additionalInfo != null && additionalInfo.getLogoFileName() != null)
```